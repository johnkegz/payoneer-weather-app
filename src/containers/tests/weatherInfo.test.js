import React from "react";
import { mount } from "enzyme";
import WeatherInfo from "../weatherInfo";
import { Provider } from "react-redux";
import store from "../../redux/store";

describe("Weather information", () => {
    let wrapper;
    beforeEach(() => {
        wrapper = mount(
        <Provider store={store}>
            <WeatherInfo />
        </Provider>
        );
    });

    it("Weather information screen to match snapshot", () => {
        expect(wrapper).toMatchSnapshot();
    });

    it("Weather information screen has one div", () => {
        expect(wrapper.find("div").length).toEqual(1);
        });
});
