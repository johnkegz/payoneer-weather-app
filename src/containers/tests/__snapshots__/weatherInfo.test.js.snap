// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Weather information Weather information 1`] = `
<Provider
  store={
    Object {
      "@@observable": [Function],
      "dispatch": [Function],
      "getState": [Function],
      "replaceReducer": [Function],
      "subscribe": [Function],
    }
  }
>
  <WeatherInfo>
    <LoadingScreen>
      <div
        className="makeStyles-container-7"
      >
        loading....
      </div>
    </LoadingScreen>
  </WeatherInfo>
</Provider>
`;

exports[`Weather information Weather information screen to match snapshot 1`] = `
<Provider
  store={
    Object {
      "@@observable": [Function],
      "dispatch": [Function],
      "getState": [Function],
      "replaceReducer": [Function],
      "subscribe": [Function],
    }
  }
>
  <WeatherInfo>
    <LoadingScreen>
      <div
        className="makeStyles-container-7"
      >
        loading....
      </div>
    </LoadingScreen>
  </WeatherInfo>
</Provider>
`;
